assets/scss/components/_people.scss

```diff
@@ -0,0 +1,30 @@
+.people {
+	padding-top: 105px;
+	padding-bottom: 130px;
+
+	&__label{
+		text-transform: uppercase;
+		color: $color-orange;
+		margin-bottom: 15px;
+	}
+
+	&__description--1 {
+		    color: #9B9B9B;
+    		font-size: 20px;
+    		line-height: 26px;
+    		margin-bottom: 27px;
+	}
+
+	&__description--2 {
+		    color: #9B9B9B;
+    		font-size: 20px;
```

Comment:
You're repeating the font-size and line-height between `&__description--1` and `&__description--2`, which means I would have create a `&__description` style and add repeated styles there.